services/ec2/drivers/Ec2EbsSnapshot.py

```diff
@@ -26,4 +27,19 @@
         if len(publicSnapshotList) > 0:
             self.results['EBSSnapshotIsPublic'] = [-1, ', '.join(publicSnapshotList)]
             
+        return
+
+    def _checkDeletedVolumeSnapshotList(self):
+        snapshotData = self.ec2Client.describe_snapshots(
+            OwnerIds = ["self"],
+        )
+
+        deletedVolumeSnapshotList = []
+        for snapshot in snapshotData['Snapshots']:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

To create a `list`, try to use `list` comprehension instead of a loop. List comprehension is the preferred way to make a list using Python, and it&#39;s simpler and easier to understand than using a loop.

[Learn more](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions)